src/tests/ftest/control/dmg_pool_query_ranks.py

```diff
@@ -119,13 +120,13 @@ def test_pool_query_ranks_mgmt(self):
         self.log_step(f"Stopping rank:{dead_rank} all_ranks={all_ranks}")
         self.server_managers[0].stop_ranks([dead_rank], self.d_log)
 
-        self.log_step(f"Waiting for pool rank {dead_rank} to be deaded")
+        self.log_step(f"Waiting for pool rank {dead_rank} to be dead")
```

Comment:
Should this step also be verifying the disabled ranks after line 124 and after line 156 like the master branch version does [here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/control/dmg_pool_query_ranks.py#L97) and [here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/control/dmg_pool_query_ranks.py#L130)?